appinsights/client.go

```diff
@@ -14,7 +15,7 @@ type TelemetryClient interface {
 	TrackMetricTelemetry(*MetricTelemetry)
 	TrackTrace(string)
 	TrackTraceTelemetry(*TraceTelemetry)
-	TrackRequest(string, time.Time, time.Duration, string, bool)
+	TrackRequest(string, string, string, time.Time, time.Duration, string, bool)
```

Comment:
I don't disagree.  But I'm trying not to break any existing (although probably limited atm) compatibility with this change.  I have a bunch of stuff slated for [the next one](https://github.com/jjjordanmsft/ApplicationInsights-Go/commit/396f92f9b96d2f20c1386b0f979e43c8c2e40771#diff-f7b8364e0b545497486543c5f91217c9R9)